### ISSUE TITLE
Update the summary for the guides

### ISSUE THREAD

**Issue by chayev** [gothamhq/docs.gothamhq.com#8]
File: <./src/content/gotham/quickstart/_index.md>

We should change what populated the summary text for the QuickStart index - it currently uses the default behavior, which isn't useful in this case.

![QuickStart Index](https://user-images.githubusercontent.com/18604081/90444592-1b957d80-e0ac-11ea-8b53-a9603c9980b8.png)

I think we should define the summary in the front matter of the articles. Thoughts?

**Comment by felicianotech** [comment]
I agree but perhaps instead of a summary, a description? Though I think Hugo/Gotham has a built-in Summary concept so it might make more sense to just lean into that.

**Comment by chayev** [comment]
We should explore both. I prefer using a built-in feature where possible.
